simulation/behavior_tree_plugin/src/vehicle/follow_trajectory_sequence/follow_clothoid_trajectory_action.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 TIER IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <behavior_tree_plugin/vehicle/follow_trajectory_sequence/follow_clothoid_trajectory_action.hpp>
+
+namespace entity_behavior
+{
+namespace vehicle
+{
+auto FollowClothoidTrajectoryAction::calculateWaypoints()
+  -> const traffic_simulator_msgs::msg::WaypointsArray
+{
+  return traffic_simulator_msgs::msg::WaypointsArray();
+}
+
+auto FollowClothoidTrajectoryAction::calculateObstacle(
+  const traffic_simulator_msgs::msg::WaypointsArray &)
+  -> const std::optional<traffic_simulator_msgs::msg::Obstacle>
+{
+  return std::nullopt;
```

Comment:
[7c621d6](https://github.com/tier4/scenario_simulator_v2/pull/906/commits/7c621d69ddfe082b450c987149f48da12acbecc5) fixes this.

Reply:
That class is a placeholder. I removed the class because I couldn't think of a reasonable TODO comment to give to the placeholder class.

Reply:
The result of the calculateWaypoints function is currently used for visualization and may be used for other applications in the future.
A TODO comment is needed so that the implementation of this function is not forgotten in the future.

Reply:
Sorry, I found this file was deleted.